addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -3312,34 +3312,6 @@ public Action:MakeBoss(Handle:timer, any:boss)
 		}
 	}
 
-	entity=-1;
-	while((entity=FindEntityByClassname2(entity, "tf_wearable_demoshield"))!=-1)
-	{
-		if(IsBoss(GetEntPropEnt(entity, Prop_Send, "m_hOwnerEntity")))
-		{
-			TF2_RemoveWearable(client, entity);
-		}
-	}
-
-	entity=-1;
-	while((entity=FindEntityByClassname2(entity, "tf_usableitem"))!=-1)
```

Comment:
Ok, then which action slot item list should we keep?  I forget why the two are different.